### ISSUE TITLE
Doesn't correctly interpret various modelines

### ISSUE THREAD

**Issue by gcp** [ctlajoie/vscode-modelines#1]
Either
/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
or
/* vim: set ts=8 sts=2 et sw=2 tw=80: */

seem to cause tabsize to be set to 8. I'm not sure VSCode's indentation model is advanced enough to properly map this, but 8 is almost certainly not what you want in this case.

**Comment by cblp** [comment]
I always want the tab size to be 8. But indentation should be 2 in this case, I think.

**Comment by gcp** [comment]
Yes, that's what I mean, too. Tabs are 8 spaces, but indentation should be 2. It seems you can only get the right indentation by changing tabSize, though (for now?).

**Comment by ctlajoie** [comment]
@gcp Since VSCode itself is not able to differentiate between tab size and indentation, this plugin is quite limited in functionality. I see that MS has backlogged the issue, so it doesn't appear to be a high priority to them.
